sphinx/source/docs/user_guide/interaction.rst

```diff
@@ -290,3 +290,26 @@ interactions such as a box zoom, wheel scroll or pan.
 .. |figure| replace:: :func:`~bokeh.plotting.figure`
 
 .. |bokeh.plotting| replace:: :ref:`bokeh.plotting <bokeh.plotting>`
+
+.. _userguide_interaction_actions_in_python:
+
+CustomJS with a Python function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A CustomJS callback can also be implemented as a Python function, which
+is then transpiled to JavaScript using PyScript. This makes it easier
+for users to define client-side interactions without having to learn
+JavaScript. To use this functionality you need the Flexx library
+(install with ``pip install flexx``). Note that this functionality is
+currently only supported on Python 3.x.
+
+For more information about the subset of Python that is supported in
+callbacks, see the `PyScript documentation
+<http://flexx.readthedocs.org/en/latest/pyscript/index.html>`_. 
```

Comment:
Use a "footnote" style reference. See, e.g., all the references here: https://raw.githubusercontent.com/bokeh/bokeh/master/sphinx/source/index.rst